polus-feature-extraction-plugin/src/main.py

```diff
@@ -1234,13 +1234,16 @@ def main():
     if segDir and intDir:
         if len(files_seg) == 0 and len(files_int)==0 :
             raise ValueError("Could not find files matching filepattern")
+        if len(files_seg) != len(files_int) :
+            raise ValueError("Number of labeled/segmented images is not equal to number of intensity images")
```

Comment:
I think this is a bad approach to handling this. There may be situations where an image collection has a variety of color channels, one of which is nuclei, and a set of nuclear segmentations. Why not just loop over the segmented images (if the values are provided) and if there isn't a matching intensity file, show a warning instead of throwing an error?